Tests/InfluxDBSwiftTests/QueryAPITests.swift

```diff
@@ -128,4 +176,63 @@ final class QueryAPITests: XCTestCase {
 
         waitForExpectations(timeout: 1, handler: nil)
     }
+
+    func testParameterizedQueryRaw() {
+        let expectation = self.expectation(description: "Success response from API doesn't arrive")
+        expectation.expectedFulfillmentCount = 2
+
+        let csv = """
+                  #datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,string,string,string,string,long,long,string
+                  #group,false,false,true,true,true,true,true,true,false,false,false
+                  #default,_result,,,,,,,,,,
+                  ,result,table,_start,_stop,_field,_measurement,host,region,_value2,value1,value_str
+                  ,,0,1677-09-21T00:12:43.145224192Z,2018-07-16T11:21:02.547596934Z,free,mem,A,west,121,11,test
+
+                  """
+
+        let responseBody = csv.data(using: .utf8)!
+
+        MockURLProtocol.handler = { request, bodyData in
+            XCTAssertEqual("Token my-token", request.allHTTPHeaderFields!["Authorization"])
+            XCTAssertEqual("application/json; charset=utf-8", request.allHTTPHeaderFields!["Content-Type"])
+            XCTAssertEqual("379", request.allHTTPHeaderFields!["Content-Length"])
+            XCTAssertEqual("identity", request.allHTTPHeaderFields!["Content-Encoding"])
+            XCTAssertEqual("identity", request.allHTTPHeaderFields!["Accept-Encoding"])
+            XCTAssertEqual("text/csv", request.allHTTPHeaderFields!["Accept"])
+            XCTAssertEqual(
+                    "\(Self.dbURL())/api/v2/query?org=my-org",
+                    request.url?.description)
+            let query = try CodableHelper.decode(Query.self, from: bodyData!).get()
+
+            XCTAssertEqual("from(bucket: params.bucketParam) |> range(start: duration(v: params.startParam))",
+                           query.query)
+            XCTAssertEqual([
```

Comment:
Please add assert to `query` params`. Something like:

```suggestion
            XCTAssertEqual(["bucketParam": "my-bucket", "startParam": "-1h"], query.params)
            XCTAssertEqual([
```